gyucheol/2일차/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+    let answer = '';
+    if((s.length % 2) == 1){
+        answer = s[((s.length + 1)/2) - 1]
+    }else{
+        answer = s[s.length/2 - 1] + s[s.length / 2]
+    } 
+    return answer;
```

Comment:
```suggestion
    return s.length % 2 === 1 ? s.slice(s.length/2, s.length/2+1) : s.slice(s.length/2-1, s.length/2+1);
```
두 가지의 조건만 있을 경우 변수 선언 없이 삼항 연산자로도 작성할 수 있습니다.
조건문에 작성하는 '=='의 경우 '==='로 작성해 주는 것이 보다 의도한 코드로 작성됩니다.

Reply:
문자열 가공하는 slice와 splice 메서드를 공부해보시면 좋을 거 같아요 ㅎㅎ